Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -84,6 +88,32 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             );
         },
 
+        _isActionDisabled: function (actionId) {
+            var actionFound;
+
+            actionFound = Y.Array.find(this.view.get('actionsList'), function (actionView) {
+                return actionView.get('actionId') === actionId;
+            });
+
+            Assert.isTrue(
+                !!actionFound.get('disabled'),
+                'The ' + actionId + ' action should be disabled for this view'
+            );
+        },
+
+        _isActionEnabled: function (actionId) {
```

Comment:
Nitpick: You could even pass an `exepected` param and use Assert.areSame to avoid having 2 method almost identical.